scripts/pretraining/pretraining_utils.py

```diff
@@ -190,7 +194,7 @@ def convert_to_npz(all_features, output_file=None):
         npz_outputs['valid_lengths'] = np.array(valid_lengths, dtype='int32')
         np.savez_compressed(output_file, **npz_outputs)
         logging.info("Saved {} features in {} ".format(len(all_features), output_file))
-    return (input_ids, segment_ids, valid_lengths)
+    return input_ids, segment_ids, valid_lengths
```

Comment:
The return conflicts with the call https://github.com/dmlc/gluon-nlp/blob/9e268c0e5e0393e4ce19a4161ef2a2f8eb56797d/scripts/pretraining/pretraining_utils.py#L86

Reply:
I think there is no need to surround it with a tuple and return. Basically, adding tuple/not is equivalent.

Reply:
Yeah that is true